ambry-api/src/main/java/com/github/ambry/config/MySqlAccountServiceConfig.java

```diff
@@ -72,13 +81,13 @@
   public final boolean updateDisabled;
 
   public MySqlAccountServiceConfig(VerifiableProperties verifiableProperties) {
-    dbUrl = verifiableProperties.getString(DB_URL, "");
-    dbUser = verifiableProperties.getString(DB_USER, "");
-    dbPassword = verifiableProperties.getString(DB_PASSWORD, "");
-    updaterPollingIntervalMs =
-        verifiableProperties.getIntInRange(UPDATER_POLLING_INTERVAL_MS_KEY, 2 * 1000, 0, Integer.MAX_VALUE);
-    updaterShutDownTimeoutMs =
-        verifiableProperties.getIntInRange(UPDATER_SHUT_DOWN_TIMEOUT_MS_KEY, 5 * 1000, 1, Integer.MAX_VALUE);
+    dbUrl = verifiableProperties.getString(DB_URL);
+    dbUser = verifiableProperties.getString(DB_USER);
+    dbPassword = verifiableProperties.getString(DB_PASSWORD);
+    updaterPollingIntervalSeconds =
+        verifiableProperties.getIntInRange(UPDATER_POLLING_INTERVAL_SECONDS, 60, 0, Integer.MAX_VALUE);
```

Comment:
What would value of 0 mean?  Update continuously?

Reply:
Used value to 0 to actually disable periodic polling. We won't start scheduler thread if config.updaterPollingIntervalSeconds is 0.